File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.swift
func solution(_ arr:[Int]) -> [Int] {
    if arr.count < 2 { return [-1] }
    else {
        var result: [Int] = []
        let m = arr.min()
        for n in arr { if n != m { result.append(n) } }
        return result 
    }
}

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        int[] answer = {};
        if (arr.length == 1) {
            answer = new int[1];
            answer[0] = -1;
        } else {
            answer = new int[arr.length-1];
            int min_num = Arrays.stream(arr).min().getAsInt();
            List<Integer> temp = new ArrayList<>();
            for(int i=0; i<arr.length; i++) {
                if (arr[i] == min_num) continue;
                temp.add(arr[i]);
            }
            for(int i=0; i<arr.length-1; i++) answer[i] = temp.get(i);
        }
        return answer;
    }
}